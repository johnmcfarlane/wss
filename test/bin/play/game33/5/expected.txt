 72  1,1  - BUNKIE
 57  1,3  - BUNKIE
 52  7,3  - BARIE
 45  2,1  - KUNBI
 40  3,1  - BAKU
 39  2,1  - AKEBI
 39  2,1  - BAKIE
 39  3,1  - KUBA
 38  5,3  - KEBAR
 37  2,1  - NIKAU
 36  3,1  - BAKE
 36  3,1  - BIKE
 36  3,1  - KIBE
 36  5,3  | KNUB
 36  2,1  - KUNAI
 35  5,3  | KNAB
 35  3,1  - NUBIA
 33  2,1  - ABUNE
 33  2,1  - BIUNE
 33  2,1  - KENAI
 33  2,1  - NUBIA
 33  3,1  - NUKE
 32  3,3  - BAKEN
 32  4,3  - KNUB
 31  3,1  - BENU
 31  8,3  - ERIK
 31  5,3  - KAB
 31  5,3  - KEB
 31  3,1  - NABU
 30  3,1  - BUNA
 30  3,1  - ENKI
 30  3,1  - KANE
 30  3,1  - KINA
 30  3,1  - KINE
 30  4,3  - KNAB
 30  3,1  - KUEI
 30  3,1  - NAKE
 30  3,1  - NIKE
 30  3,1  - NUBA
 30  3,1  - UNBE
 28  6,3  - ABIR
 28  3,1  - BEAU
 28  5,4  | KAB
 28  5,4  | KEB
 28  5,3  | NAIB
 27  3,1  - AUBE
 27  3,1  - BANE
 27  3,1  - BANI
 27  7,3  - BARI
 27  3,1  - BENA
 27  3,1  - BENI
 27  7,3  - BERI
 27  3,1  - BINE
 27  5,5  | KB
 27  5,3  - NUB
 26  5,4  | NUB
 25  3,3  - BAKU
 25  5,3  - NAB
 25  5,4  | NAB
 25  5,3  - NEB
 25  5,4  | NEB
 25  5,4  | NIB
 24  3,1  - ABIE
 24  7,3  - BAR
 24  7,3  - BER
 24  5,4  - BUNKS
 24  4,3  - IKAN
 24  3,3  - KUBA
 23  5,5  | AB
 23  3,3  - BAKE
 23  5,4  - BANKS
 23  3,3  - BIKE
 23  5,3  - BUN
 23  9,1  - HUK
 23  5,5  | IB
 23  5,3  - KEN
 23  3,3  - KIBE
 23  9,3  - RIK
 22  3,1  - AINU
 22  4,1  - BAUK
 22  5,4  - BEAKS
 22  8,3  - ERIN
 22  4,3  - IBAN
 22  4,1  - KNUB
 22  4,3  - KUAN
 21  6,3  - AB
 21  3,1  - AUNE
 21  5,3  - BAN
 21  4,3  - BAUN
 21  5,3  - BEN
 21  4,4  - BUENAS
 21  9,1  - HAK
 21  3,1  - NIUE
 21  3,1  - UNAI
 21  3,1  - UNIE
 20  4,1  - BEAK
 20  7,4  - BKS
 20  9,1  - HUB
 20  4,1  - KNAB
 20  4,1  - NEUK
 20  3,3  - NUKE
 20  8,2  | PAU
 20  9,3  - RIB
 19  2,3  - ABUNE
 19  6,4  - AUKS
 19  4,3  - BEAN
 19  3,3  - BENU
 19  4,3  - BIEN
 19  8,3  - ERIA
 19  6,4  - INKS
 19  6,4  - KABS
 19  3,3  - KANE
 19  3,3  - KINA
 19  3,3  - KINE
 19  3,3  - NABU
 19  3,3  - NAKE
 19  7,3  - NERI
 19  3,3  - NIKE
 18  3,1  - AINE
 18  5,1  - AUK
 18  5,6  - BOOZES
 18  3,3  - BUNA
 18  9,1  - HAB
 18  5,1  - KAB
 18  5,1  - KEB
 18  7,4  | KIOEA
 18  4,1  - NAIK
 18  3,3  - NUBA
 18  3,3  - UNBE
 17  4,3  - AKU
 17  2,3  - BAKIE
 17  3,3  - BANE
 17  3,3  - BEAU
 17  3,3  - BENA
 17  3,3  - BINE
 17  8,4  - KSI
 17  5,1  - NAK
 17  5,1  - NEK
 17  2,3  - NIKAU
 17  5,1  - NUB
 17  4,4  - NUBIAS
 17  6,4  - NUBS
 17  4,4  - UNBIAS
 16  5,1  - AIK
 16  3,3  - AUBE
 16  7,1  | BIKE
 16  2,3  - BIUNE
 16  7,4  - BUSK
 16  6,4  - NABS
 16  4,1  - NAIB
 16  7,3  - NAR
 16  6,4  - NEBS
 16  6,4  - NIBS
 16  2,3  - NUBIA
 16  4,3  - UKE
 15  7,4  - ABSI
 15  4,3  - ABU
 15  4,3  - AKE
 15  5,4  - ANKUS
 15  4,4  - ANUBIS
 15  5,3  - EAN
 15  5,3  - IAN
 15  4,3  - IKE
 15  7,1  | KIBE
 15  7,4  - KUSAN
 15  5,1  - NAB
 15  5,1  - NEB
 15  5,1  - NIB
 15  6,3  - UN
 14  7,4  - ABS
 14  7,1  - AUH
 14  5,4  - BAKES
 14  5,4  - BANUS
 14  7,4  - BASKE
 14  5,4  - BIKES
 14  7,1  | BINK
 14  5,3  - BU
 14  9,1  - HUN
 14  5,3  - KA
 14  5,4  - KAINS
 14  5,4  - KIBES
 14  5,1  - KUA
 14  7,1  | NIKE
 14  8,2  | PEA
 14  9,3  - RIN
 13  6,1  - AB
 13  6,4  - ABUSE
 13  3,3  - AINU
 13  6,3  - AN
 13  7,4  - BASK
 13  5,4  - BEANS
 13  5,4  - BEAUS
 13  5,4  - BIENS
 13  7,4  - BISK
 13  6,4  - BUNS
 13  6,3  - EN
 13  6,1  - IB
 13  4,3  - IBA
 13  6,4  - KANS
 13  6,4  - KENS
 13  5,4  - KINASE
 13  6,4  - KINS
 13  6,1  | KNUB
 13  4,3  - KUA
 13  4,3  - KUE
 13  7,4  - KUSA
 13  3,3  - NIUE
 13  5,4  - NUKES
 13  5,4  - UNBASE
 12  3,3  - ABIE
 12  7,1  | AIK
 12  4,1  - AKU
 12  8,1  | APE
 12  3,3  - AUNE
 12  5,3  - BA
 12  6,4  - BANS
 12  5,3  - BE
 12  6,4  - BENS
 12  6,4  - BINS
 12  8,1  | EPA
 12  9,1  - HAN
 12  9,1  - HAU
 12  9,1  - HEN
 12  9,1  - HIN
 12  7,2  | IKE
 12  5,4  - KANES
 12  5,1  - KEA
 12  5,4  - KINES
 12  7,4  - KISAN
 12  6,1  | KNAB
 12  6,4  - UKASE
 12  3,3  - UNIE
 11  4,1  - ABU
 11  3,3  - AINE
 11  5,4  - AINUS
 11  5,4  - BANES
 11  7,4  - BASIN
 11  5,1  - BEA
 11  7,4  - BESAN
 11  7,4  - BESIN
 11  5,4  - BINES
 11  7,4  - BUS
 11  9,1  - HUE
 11  9,1  - HUI
 11  5,4  - IANUS
 11  7,4  - INSUE
 11  4,3  - KAE
 11  4,3  - KEA
 11  4,3  - KIE
 11  7,1  | KINA
 11  7,1  | KINE
 11  4,1  - KUA
 11  4,1  - KUE
 11  6,4  - KUES
 11  4,1  - KUI
 11  9,5  - LUB
 11  5,4  - NABIS
 11  7,1  | NIB
 11  9,3  - RIA
 11  9,3  - RIE
 11  8,1  - UH
 11  4,1  - UKE
 11  6,4  - UKES
 11  8,1  | UP
 10  5,4  - ABIES
 10  4,1  - AKE
 10  7,4  - ANSU
 10  6,4  - ANUS
 10  4,3  - BAE
 10  4,3  - BEA
 10  6,4  - BEISA
 10  7,1  | BINE
 10  7,1  | BINI
 10  5,1  - BU
 10  7,2  | IBA
 10  4,1  - IKE
 10  7,2  | INK
 10  7,4  - INSEA
 10  4,1  - KAE
 10  6,4  - KAES
 10  4,1  - KAI
 10  4,1  - KEA
 10  6,4  - KEAS
 10  4,1  - KIE
 10  7,1  | KIN
 10  9,5  - LAK
 10  9,5  - LEK
 10  5,3  - NU
 10  4,1  - UBI
 10  4,3  - UNA
  9  6,4  - AINS
  9  4,3  - ANE
  9  7,4  - ANSI
  9 10,1  | ASE
  9  8,4  - ASK
  9  4,1  - BAE
  9  4,1  - BAI
  9  6,4  - BAIS
  9  7,4  - BASE
  9  4,1  - BEA
  9  7,4  - BESA
  9  6,4  - BIAS
  9  7,1  | BIEN
  9  7,1  | BIN
  9  7,4  - BISE
  9  4,3  - EAU
  9  6,1  | ENKI
  9  9,1  - HAE
  9  9,1  - HEI
  9  9,1  - HIA
  9  9,1  - HIE
  9  7,2  | IB
  9  4,1  - IBA
  9  8,4  - ISBA
  9 10,1  | ISE
  9  5,1  - KA
  9  7,4  - KAS
  9  7,4  - KES
  9  5,1  - KI
  9  9,5  - LAB
  9  9,5  - LIB
  9  5,1  - NEA
  9  7,4  - NUS
  9  5,4  - UNAIS
  9  6,1  | UNBE
  9  7,4  - UNS
  8  7,1  | AINU
  8  6,4  - ANISE
  8  7,4  - ANS
  8  8,4  - ASB
  8  7,4  - AUS
  8  5,1  - BA
  8  7,4  - BAS
  8  5,1  - BE
  8  5,1  - BI
  8  7,4  - BIS
  8  4,1  - EAU
  8  7,4  - ENS
  8  6,1  | INK
  8  7,4  - INS
  8  7,1  | KIE
  8  9,5  - LUE
  8  9,5  - LUI
  8  5,3  - NA
  8  4,3  - NAE
  8  6,2  | NAK
  8  5,3  - NE
  8  4,3  - NEA
  8  6,2  | NEK
  8  7,1  | NIUE
  8  5,1  - NU
  8  6,2  | NUB
  8  9,4  - SUK
  8  4,1  - UNA
  8  4,1  - UNI
  8  8,4  - USE
  7  7,1  | AINE
  7  4,1  - ANE
  7  6,4  - ANES
  7  4,1  - ANI
  7  6,4  - ANIS
  7  8,3  - AR
  7  7,4  | EBOE
  7  8,3  - ER
  7  8,4  - ESAU
  7  9,5  - LAN
  7  9,5  - LEN
  7  9,5  - LEU
  7  9,5  - LIN
  7  9,5  - LU
  7  6,2  | NAB
  7  4,1  - NAE
  7  6,4  - NAIS
  7  7,4  - NASI
  7  4,1  - NEA
  7  6,2  | NEB
  7  4,1  - NEI
  7  6,4  - NIAS
  7  6,2  | NIB
  7  6,4  - NIES
  7  8,2  | PA
  7  8,2  | PE
  7  9,4  - SAK
  7 10,2  | SEI
  7  9,4  - SKI
  7  9,4  - SUB
  7  8,4  - US
  7 10,5  | US
  6  5,3  - AE
  6  7,1  | AIN
  6  6,1  | ANNE
  6  6,1  | ANNI
  6 10,1  | AS
  6  7,5  | BOE
  6 10,1  | ES
  6  8,4  - ESU
  6  9,1  - HA
  6  9,1  - HE
  6  9,1  - HI
  6  6,1  | INNE
  6 10,1  | IS
  6  8,4  - ISN
  6  7,1  | KI
  6  9,5  - LAI
  6  9,5  - LEA
  6  9,5  - LEI
  6  9,5  - LIE
  6  5,1  - NA
  6  5,1  - NE
  6  7,4  - NIS
  6  6,2  | NUN
  6  9,4  - SAB
  6  9,4  - SIB
  6  6,1  | UNAI
  6  6,1  | UNIE
  6  6,1  | UNN
  5  5,1  - AE
  5  7,4  - AES
  5  5,1  - AI
  5  7,4  - AIS
  5  6,1  | ANN
  5 10,5  | AS
  5  8,4  - ASE
  5  7,1  | BI
  5  7,2  | IAN
  5  7,2  | IN
  5  6,1  | INN
  5  8,4  - ISE
  5  9,5  - LA
  5  6,2  | NAN
  5  9,4  - SUN
  5  6,1  | UNA
  5  6,1  | UNI
  4  6,1  | ANE
  4  6,1  | ANI
  4  8,4  - AS
  4  8,4  - ES
  4  8,4  - IS
  4  6,2  | NAE
  4  6,2  | NEA
  4  6,2  | NEI
  4  6,2  | NU
  4  9,4  - SAN
  4  9,4  - SAU
  4  9,4  - SEN
  4  9,4  - SIN
  4  9,4  - SUE
  4  9,4  - SUI
  4  6,1  | UN
  3  6,1  | AN
  3  6,1  | EN
  3  6,1  | IN
  3  6,2  | NA
  3  6,2  | NE
  3  9,4  - SAE
  3  9,4  - SAI
  3  9,4  - SEA
  3  9,4  - SEI
  3  9,4  - SIA
  3  9,4  - SIE
  2  7,1  | AI
  2  9,4  - SI
