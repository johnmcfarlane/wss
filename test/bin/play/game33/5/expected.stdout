 38  5,3  - KEBAR
 36  3,1  - BAKE
 36  3,1  - BIKE
 36  3,1  - KIBE
 35  3,1  - NUBIA
 33  3,1  - KUNA
 33  3,1  - KUNE
 33  2,1  - NUBIA
 33  3,1  - NUKE
 31  5,3  - KAB
 30  3,1  - BUNA
 30  3,1  - KANE
 30  3,1  - KINA
 30  3,1  - KINE
 30  3,1  - UNBE
 28  3,1  - BEAU
 28  5,4  | KAB
 27  3,1  - BANE
 27  3,1  - BANI
 27  3,1  - BINE
 27  3,1  - NABE
 27  5,3  - NUB
 26  5,4  | NUB
 25  5,3  - NAB
 25  5,4  | NAB
 25  5,3  - NEB
 25  5,4  | NEB
 25  5,4  | NIB
 24  7,3  - BAR
 24  5,4  - BUNKS
 23  5,5  | AB
 23  3,3  - BAKE
 23  5,4  - BANKS
 23  3,3  - BIKE
 23  5,3  - BUN
 23  5,5  | IB
 23  5,3  - KEN
 23  3,3  - KIBE
 22  5,4  - BEAKS
 21  6,3  - AB
 21  5,3  - BAN
 21  5,3  - BEN
 21  5,4  - NEUKS
 21  3,1  - UNAI
 20  4,1  - BEAK
 20  9,1  - HUB
 20  3,3  - KUNA
 20  3,3  - KUNE
 20  4,1  - NEUK
 20  3,3  - NUKE
 20  9,3  - RIB
 19  6,4  - AUKS
 19  4,3  - BEAN
 19  6,4  - INKS
 19  6,4  - KABS
 19  3,3  - KANE
 19  3,3  - KINA
 19  3,3  - KINE
 18  5,1  - AUK
 18  5,6  - BOOZES
 18  3,3  - BUNA
 18  5,1  - KAB
 18  3,3  - UNBE
 17  3,3  - BANE
 17  3,3  - BEAU
 17  3,3  - BINE
 17  3,3  - NABE
 17  5,1  - NUB
 17  4,4  - NUBIAS
 17  6,4  - NUBS
 16  7,1  | BIKE
 16  7,4  - BUSK
 16  6,4  - NABS
 16  6,4  - NEBS
 16  6,4  - NIBS
 16  2,3  - NUBIA
 16  4,3  - UKE
 15  5,4  - ANKUS
 15  7,1  | KIBE
 15  5,1  - NAB
 15  5,1  - NEB
 15  5,1  - NIB
 15  6,3  - UN
 14  7,4  - ABS
 14  5,4  - BAKES
 14  5,4  - BIKES
 14  9,1  - HUN
 14  5,3  - KA
 14  5,4  - KAINS
 14  5,4  - KIBES
 14  8,2  | PEA
 14  9,3  - RIN
 13  6,1  - AB
 13  6,4  - ABUSE
 13  6,3  - AN
 13  7,4  - BASK
 13  5,4  - BEANS
 13  5,4  - BEAUS
 13  7,4  - BISK
 13  6,4  - BUNS
 13  6,3  - EN
 13  6,1  - IB
 13  6,4  - KENS
 13  5,4  - KINASE
 13  6,4  - KINS
 13  4,3  - KUE
 13  5,4  - NUKES
 12  8,1  | APE
 12  5,3  - BA
 12  6,4  - BANS
 12  5,3  - BE
 12  6,4  - BENS
 12  6,4  - BINS
 12  5,4  - BUNAS
 12  9,1  - HEN
 12  9,1  - HIN
 12  5,4  - KANES
 12  5,1  - KEA
 12  5,4  - KINAS
 12  5,4  - KINES
 12  6,4  - UKASE
 11  5,4  - BANES
 11  7,4  - BASIN
 11  5,4  - BINES
 11  7,4  - BUS
 11  9,1  - HUE
 11  4,3  - KAE
 11  4,3  - KEA
 11  7,1  | KINA
 11  7,1  | KINE
 11  4,1  - KUE
 11  6,4  - KUES
 11  5,4  - NABES
 11  5,4  - NABIS
 11  7,1  | NIB
 11  9,3  - RIA
 11  8,1  - UH
 11  4,1  - UKE
 11  6,4  - UKES
 11  8,1  | UP
 10  6,4  - ANUS
 10  7,1  | BINE
 10  7,2  | INK
 10  4,1  - KAE
 10  6,4  - KAES
 10  4,1  - KEA
 10  6,4  - KEAS
 10  7,1  | KIN
 10  9,5  - LEK
 10  5,3  - NU
  9  6,4  - AINS
  9  4,3  - ANE
  9  8,4  - ASK
  9  7,4  - BASE
  9  6,4  - BIAS
  9  7,1  | BIN
  9  7,4  - BISE
  9  4,3  - EAU
  9  9,1  - HAE
  9  9,1  - HIE
  9  7,2  | IB
  9  8,4  - ISBA
  9  5,1  - KA
  9  7,4  - KAS
  9  5,1  - KI
  9  7,4  - KIS
  9  9,5  - LAB
  9  9,5  - LIB
  9  7,4  - NUS
  9  5,4  - UNAIS
  9  6,1  | UNBE
  9  7,4  - UNS
  8  6,4  - ANISE
  8  5,1  - BA
  8  7,4  - BAS
  8  5,1  - BE
  8  7,4  - BES
  8  5,1  - BI
  8  7,4  - BIS
  8  4,1  - EAU
  8  7,4  - ENS
  8  6,1  | INK
  8  7,4  - INS
  8  5,3  - NA
  8  4,3  - NAE
  8  5,3  - NE
  8  5,1  - NU
  8  6,2  | NUB
  8  9,4  - SUK
  8  8,4  - USE
  7  4,1  - ANE
  7  6,4  - ANES
  7  4,1  - ANI
  7  6,4  - ANIS
  7  8,3  - AR
  7  9,5  - LEU
  7  9,5  - LIN
  7  6,2  | NAB
  7  4,1  - NAE
  7  6,2  | NEB
  7  6,2  | NIB
  7  8,2  | PA
  7 10,2  | SEI
  7  9,4  - SKA
  7  9,4  - SKI
  7  9,4  - SUB
  7  8,4  - US
  6  5,3  - AE
  6  7,1  | AIN
  6 10,1  | AS
  6 10,1  | ES
  6  9,1  - HA
  6  9,1  - HE
  6  9,1  - HI
  6 10,1  | IS
  6  7,1  | KI
  6  9,5  - LEA
  6  9,5  - LEI
  6  9,5  - LIE
  6  5,1  - NA
  6  5,1  - NE
  6  6,2  | NUN
  6  9,4  - SAB
  6  9,4  - SIB
  6  6,1  | UNAI
  5  5,1  - AE
  5  5,1  - AI
  5  7,4  - AIS
  5 10,5  | AS
  5  7,1  | BI
  5  7,2  | IN
  5  6,1  | INN
  5 10,5  | IS
  5  9,5  - LA
  5  9,5  - LI
  5  6,2  | NAN
  5  9,4  - SUN
  4  6,1  | ANE
  4  6,1  | ANI
  4  8,4  - AS
  4  8,4  - ES
  4  8,4  - IS
  4  6,2  | NAE
  4  6,2  | NU
  4  9,4  - SAU
  4  9,4  - SEN
  4  9,4  - SIN
  4  9,4  - SUE
  4  6,1  | UN
  3  6,1  | AN
  3  6,1  | EN
  3  6,1  | IN
  3  6,2  | NA
  3  6,2  | NE
  3  9,4  - SAE
  3  9,4  - SEA
  3  9,4  - SEI
  2  7,1  | AI
  2  9,4  - SI
